### PR TITLE
feat(functions): columns function

### LINKED ISSUE #626: Create a columns function
Create a function called `columns` that will produce a table that has one row for each column in the input table, whose name is placed in the `_value` column.

This is to replace functionality that is lost by fixing `keys` to produce a more intuitive result, per this issue: https://github.com/influxdata/flux/issues/625

### PR DESCRIPTION
**Pull request by affo**
Fixes #626 

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

**Comment by affo** [comment]
For the weird final call to `Do`, see #643

**Comment by wolffcm** [comment]
Looks good to me.  @nathanielc may wish to review the change to the SPEC for `columns` as well as `keys` (#641)

File: docs/SPEC.md [hunk 1]
```diff
@@ -2144,6 +2144,32 @@ from(bucket: "telegraf/autogen")
 Records are grouped into a single table.  
 The group key of the resulting table is empty.
 
+#### Columns
+
+Columns lists the column labels of input tables.
+For each input table, it outputs a table with the same group key columns, plus a `_value` column containing the labels of the input table's columns.
+Each row in an output table contains the group key value and the label of one column of the input table.
+So, each output table has the same number of rows as the number of columns of the input table.
```
Comment (nathanielc) [inline docs/SPEC.md]: This looks great. The `_value` should be a parameter to the function with a default value of `_value.

Comment (affo) [reply docs/SPEC.md]: Ok, shouldn't do the same for `keys`?

**Comment by nathanielc** [comment]
Make sure to use `BREAKING CHANGE: ` in the commit message to indicate the breaking change. Thanks!

**Pull request merged**